### PR TITLE
Add summary and documentation, change description and author

### PR DESCRIPTION
**Pull request by pmaier1**
@hurradieweltgehtunter 
We might want to embed the video?

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -2,12 +2,30 @@
...
+Guest accounts are grouped under a virtual group allowing administrators to easily filter them in the user management.
+Furthermore guests are fully auditable with the [ownCloud Auditing application](https://marketplace.owncloud.com/apps/admin_audit).
+
+### Resources
+- Video: [ownCloud X Insights: Guest Users](https://www.youtube.com/watch?v=L42PBHgqKVI)
 	</description>
 	<version>0.4.1</version>
+	<documentation>
+    		<user>https://doc.owncloud.com/server/10.0/user_manual/files/webgui/sharing.html?highlight=guest#sharing-files-with-guest-users</user>
+    		<admin></admin>
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: remove empty tags

File: appinfo/info.xml [hunk 2]
```diff
@@ -2,12 +2,30 @@
...
+    		<developer></developer>
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: remove empty tags

File: appinfo/info.xml [hunk 3]
```diff
@@ -2,12 +2,30 @@
 <info>
 	<id>guests</id>
 	<name>Guests</name>
+	<summary>Share with externals easily via email address</summary>
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: "Share files with ..." ?

**Comment by hurradieweltgehtunter** [comment]
hm video embedding is possible with a new plugin but we'd need to extract it from description. For now, youtube link must be enough i think.

File: appinfo/info.xml (hunk 1 above)
Comment (pmaier1) [inline appinfo/info.xml]: From https://github.com/owncloud/contacts/pull/607#discussion_r131877338

> leave those empty where no link is available

File: appinfo/info.xml (hunk 3 above)
Comment (pmaier1) [inline appinfo/info.xml]: - it's not only about files but also folders, address books, calendars etc.
- via email address is the essential feature of the app

Comment (cortho) [inline appinfo/info.xml]: I'd ĺike to see 'email address' in the README too

**Pull request merged**